### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by KnathanM**
I'm reading through the documentation and noticed some typos in one line. Feel free to merge now if you'd like, or if I see other typos, I'll continue to add them here as a I read.

### CODE REVIEW

File: documentation/source/users/rmg/input.rst [hunk 1]
```diff
@@ -59,8 +59,8 @@ ThermoLibrary field must be with respect to the :file:`$RMG/RMG-database/input/t
 directory.
 
 .. note::
-	Checks during the initialization are maid to avoid users to use "liquid thermo librairies" in gas phase simulations or to use
-	"liquid phase libraries" obtained in another solvent that the one defined in the input file in liquid phase simulations.
+	Checks during the initialization are made to avoid users using "liquid thermo librairies" in gas phase simulations or using
```
Comment (jonwzheng) [inline documentation/source/users/rmg/input.rst:62]: "libra**i**ries" is also a typo here

**Review by jonwzheng — COMMENTED** [review]
looks good - one minor comment (looks like there's another typo in this line, amazingly)

**Comment by JacksonBurns** [comment]
I've approved the workflows to run - feel free to leave open and work or merge after addressing review comments!

**Comment by KnathanM** [comment]
I think best to leave this open until I have worked through the RMG user guide some more in case I see more typos. I was able to install RMG today. 
I'm also okay to merge this PR whenever though to get it off the list of open PRs.

**Comment by JacksonBurns** [comment]
> I think best to leave this open until I have worked through the RMG user guide some more in case I see more typos.

Sounds good to me!

> I was able to install RMG today.

Glad to hear it!

> I'm also okay to merge this PR whenever though to get it off the list of open PRs.

We have a bot that will mark this as 'stale' if it isn't touched for 90 consecutive days - if that comes to pass, we just merge it then. Otherwise fine to leave it open - we usually have pretty long-term PRs around here.

**Comment by jonwzheng** [comment]
I propose we merge this in and move any further doc updates into #2704 since we are actively combing through the docs in that PR

**Review by jonwzheng — APPROVED** [review]

**Comment by KnathanM** [comment]
sounds good to me

**Pull request merged**